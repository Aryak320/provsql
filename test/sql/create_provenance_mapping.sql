\set ECHO none
\pset format unaligned
SET search_path TO provsql_test,provsql;

SELECT create_provenance_mapping('Personnel_Id','personnel','id');
SELECT value FROM Personnel_Id;
DROP TABLE Personnel_Id;

SELECT create_provenance_mapping('Personnel_Id','personnel','id','t');
SELECT value FROM "Personnel_Id";
DROP TABLE "Personnel_Id";
