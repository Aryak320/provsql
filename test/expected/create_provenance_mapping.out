\set ECHO none
create_provenance_mapping

(1 row)
value
1
2
3
4
5
6
7
(7 rows)
create_provenance_mapping

(1 row)
value
1
2
3
4
5
6
7
(7 rows)
